src/bin/drag_and_drop_listbox.rs

```diff
@@ -0,0 +1,193 @@
+// A drag-and-drop example. You can drag the list items using the icon on the left, and drop them
+// to other icons to move the list item. Originally listbox-dnd.c in GTK+ repository. Ported to
+// gtk-rs by Ömer Sinan Ağacan <omeragacan@gmail.com>.
+
+extern crate cairo;
+extern crate gdk;
+extern crate gio;
+extern crate gtk;
+
+use gdk::{Atom, DragAction, DragContext, ModifierType, Screen};
+use gio::prelude::*;
+use gtk::*;
+use std::env::args;
+
+fn main() {
+    let app = gtk::Application::new("com.github.gtk-rs.examples.basic", Default::default())
+        .expect("Initialization failed...");
+
+    app.connect_activate(|app| {
+        build_ui(app);
```

Comment:
You want to use `startup` here instead of `activate`, see comment on the other PR